android/benchmark/src/androidTest/kotlin/dev/zacsweers/jsonserialization/android/AndroidBenchmark.kt

```diff
@@ -236,6 +244,40 @@ class AndroidBenchmark(
     }
   }
 
+  class JacksonDatabind(json: String) {
+    val mapper = JsonMapper()
+    val response: JackResponse = mapper.readValue(json, JackResponse::class.java)
+  }
+
+  class JacksonDatabindBuffer(private val json: String) {
+    val mapper = JsonMapper()
+    val response: JackResponse = mapper.readValue(json, JackResponse::class.java)
+    lateinit var bufferedInput: BufferedInputStream
+    lateinit var bufferedOutput: BufferedOutputStream
+
+    fun setupIteration() {
+      bufferedInput = BufferedInputStream(ByteArrayInputStream(json.toByteArray()))
+      bufferedOutput = BufferedOutputStream(ByteArrayOutputStream())
```

Comment:
There might be a better way to do this (and the related function in the kotlin databind class)